internal/services/load_balancer_pool/model.go

```diff
@@ -28,9 +28,9 @@ type LoadBalancerPoolModel struct {
 	LoadShedding       customfield.NestedObject[LoadBalancerPoolLoadSheddingModel]       `tfsdk:"load_shedding" json:"load_shedding,computed_optional"`
 	NotificationFilter customfield.NestedObject[LoadBalancerPoolNotificationFilterModel] `tfsdk:"notification_filter" json:"notification_filter,computed_optional"`
 	OriginSteering     customfield.NestedObject[LoadBalancerPoolOriginSteeringModel]     `tfsdk:"origin_steering" json:"origin_steering,computed_optional"`
-	CreatedOn          timetypes.RFC3339                                                 `tfsdk:"created_on" json:"created_on,computed" format:"date-time"`
+	CreatedOn          types.String                                                      `tfsdk:"created_on" json:"created_on,computed"`
```

Comment:
are we fixing this in the config? or running with a custom code option here?

Reply:
Updated in the config and it looks like it codegen'd correctly.